awswrangler/distributed/ray/_register.py

```diff
@@ -45,7 +44,10 @@ def register_ray() -> None:
             _is_pandas_or_modin_frame,
             _split_modin_frame,
         )
-        from awswrangler.distributed.ray.modin.s3._read_parquet import _read_parquet_distributed
+        from awswrangler.distributed.ray.modin.s3._read_parquet import (
```

Comment:
This is up for debate, but for me a method should only go under `distributed.ray.modin` if it's somehow specific to Modin. In this case all the method does is read a Pyarrow schema. So it should live one level higher under `distributed.ray` because it could be use for other memory formats in the future, independently of Modin

Reply:
Good point